azurerm/internal/services/batch/data_source_batch_pool.go

```diff
@@ -278,6 +278,14 @@ func dataSourceArmBatchPool() *schema.Resource {
 					},
 				},
 			},
+			"metadata": {
+				Type:     schema.TypeMap,
+				Optional: true,
```

Comment:
This should be computed for a data source

Reply:
added Computer: true